.gitignore

```
@@ -29,11 +29,11 @@ yarn-error.log*
 .env
 
 # yarn NON Zero-install
+.yarn/*
 .pnp.*
 .yarn/*
 !.yarn/patches
 !.yarn/plugins
```

```suggestion
.pnp.*
.yarn/*
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```

From https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored